crates/polars-plan/src/logical_plan/optimizer/cse_expr.rs

```diff
@@ -256,40 +256,45 @@ impl ExprIdentifierVisitor<'_> {
 
     /// return `None` -> node is accepted
     /// return `Some(_)` node is not accepted and apply the given recursion operation
+    /// `Some(_, true)` don't accept this node, but can be a member of a cse.
+    /// `Some(_,  false)` don't accept this node, and don't allow as a member of a cse.
     fn accept_node(&self, ae: &AExpr) -> Option<(VisitRecursion, bool)> {
+        // Don't allow this node, but allow as a member of a cse.
```

Comment:
I guess this comment should be swapped with the comments of `REFUSE_ALLOW_MEMBER`.

Reply:
Good catch!